### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by Bornittah**
add index.html
add style.css
add linters, 
update the readme file
add test.md and other files to gitignore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+Check out the issues tab
+
+# Citation
+
+# License
+MIT License
+
+# Author
+Agasha Bornittah 
+Github: https://github.com/Bornittah
```
Comment (ShadyShawkat) [inline README.md]: - [ ] Please don't forget to fill in those sections and have a title for the project at the top of your `README` file. Also kindly add the `Getting started` section with info on how to set up your project locally.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Bornittah, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
- [x] Kindly add `test.md` to your `.gitignore` file.
- [x] Please add a meaningful description to your pull request, you can mention the features implemented in this pull request. This helps make your PR look more professional.
- [ ] Please make sure to set up the linters for this project. You can use [these linters](https://github.com/microverseinc/linters-config/tree/master/html-css) provided by Microverse. To know how to set it up and get it running, you can read the `README` under the linters. Also, please make sure that they all are running correctly and passing with green checks. ✔️ 
- [ ] _Check the comments under the review for more changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chrystalme** [comment]
Hi @Bornittah , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
You have added 

- [ ] test.md and .gitignore
- [ ] You have setup linters correctly


## Suggested changes ♻️ 
As suggested by the first reviewer, 

- [ ]  You have not given a descriptive name to your PR. Give a descriptive name to your PR.
- [ ] Your Readme file does not look professional.

## [Optional]

- [ ] Try to remove the node_modules file from your repository. It makes your repository look more professional. Read this [article](https://www.git-tower.com/learn/git/faq/github-delete-folder) to know how.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# Hello @Bornittah  😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**